### DISCUSSION TITLE
Request for .AppImage build

### DISCUSSION THREAD

**Discussion by Belingheri** [ivangabriele/clamav-desktop#161] [Feature Request Proposals]
Now a lot of Electron project build in `.AppImage` this "build" can be execute and install in every distro linux.
I have no idea of the effort to build in `AppImage` but you can expand discro compatibility easly, and for fedora user like me we will not wait for the `.rpm` build

Thanks

**Comment by ivangabriele** [comment]
Hi,

It's an Antivirus and not a simple "app". It requires a lot of extended permissions, including installing a daemon with full sudo rights + its own dedicated group and user to handle things like quarantined infected files.

Unless I'm missing something this is the opposite of what an AppImage does, which is to protect users when installing apps by running in a "contained" way, under normal user rights.

So in short I believe that to be impossible, not because Tauri can't (it's not under Electron anymore and I don't advise you to download what you find in the releases, they barely work and are outdated), but because it's too restrictive to run an app requiring full admin privileges.
